dissect/target/filesystems/zip.py

```diff
@@ -0,0 +1,136 @@
+import io
+import logging
+import zipfile
+
+from dissect.util.stream import BufferedStream
+
+from dissect.target.exceptions import FileNotFoundError, FilesystemError
+from dissect.target.filesystem import Filesystem, VirtualFile, VirtualFilesystem
+from dissect.target.helpers import fsutil
+
+log = logging.getLogger(__name__)
+
+
+class ZipFilesystem(Filesystem):
+    """This Zip filesystem implementation is WIP and
+    has only been tested for opening java jar archives.
+    Its structure has been copied from the tar filesystem
+    implementation.
+    """
```

Comment:
```suggestion
    """Filesystem implementation for zip files.
    
    Python does not have symlink support in the zipfile module, so that's not currently supported.
    See https://github.com/python/cpython/issues/82102 for more information.
    """
```

The untested comment is unnecessary in the docstring, but it would be nice to put the symlink warning there so you don't have to repeat it everywhere.